docs/DifferentiableProgramming.md

```diff
@@ -2016,10 +2016,11 @@ differentiating either one will provide derivatives with respect to parameters
 `T` and `U`. Here are some examples of first-order function types and their
 corresponding curried function types:
 
-| First-order function type                 | Curried function type                           |
-| @differentiable (T, U) -> V               | @differentiable (T) -> @differentiable (U) -> V |
-| @differentiable (T, @noDerivative U) -> V | @differentiable (T) -> (U) -> V                 |
-| @differentiable (@noDerivative T, U) -> V | (T) -> @differentiable (U) -> V                 |
+| First-order function type                   | Curried function type                             |
+|---------------------------------------------+---------------------------------------------------|
```

Comment:
The table still doesn't appear to be rendered correctly. I think this should fix it:
```suggestion
|---------------------------------------------|---------------------------------------------------|
```

Here's what it looks like:

| First-order function type                   | Curried function type                             |
|---------------------------------------------|---------------------------------------------------|
| `@differentiable (T, U) -> V`               | `@differentiable (T) -> @differentiable (U) -> V` |
| `@differentiable (T, @noDerivative U) -> V` | `@differentiable (T) -> (U) -> V`                 |
| `@differentiable (@noDerivative T, U) -> V` | `(T) -> @differentiable (U) -> V`                 |

Reply:
Ugh, emacs is doing weird stuff here. Thanks for pointing this out.